doc/en/reference.rst

```diff
@@ -248,6 +248,16 @@ Will create and attach a :class:`Mark <_pytest.mark.structures.Mark>` object to
     mark.args == (10, "slow")
     mark.kwargs == {"method": "thread"}
 
+Example for using multiple custom markers:
+
+.. code-block:: python
+
+    @pytest.mark.timeout(10, "slow", method="thread")
+    @pytest.mark.slow
+    def test_function():
+        ...
+
+Will order execution of markers from inside - out. The above example will execute @pytest.mark.slow before @pytest.mark.timeout(10, "slow", method="thread").
```

Comment:
Markers themselves aren't "executed" - what changes is the result of `Node.iter_markers`. Additionally, the inline code here should be marked as such, and I think it'd be a bit more readable when not repeating the arguments of the markers - thus, I'd propose something like:

```suggestion
When :meth:`Node.iter_markers <_pytest.nodes.Node.iter_markers>` is used with multiple markers, the marker closest to the function will be iterated over first. The above example will result in ``@pytest.mark.slow`` followed by ``@pytest.mark.timeout(...)``.
```

Reply:
i believe the word we are looking for is "apply"

Reply:
This makes perfect sense, thank you. 
In addition, I would add Node.iter_markers_with_node along with Node.iter_markers.